paper/paper.md

```diff
@@ -53,44 +53,56 @@ bibliography: paper.bib
+- PPIs, for mining interaction patterns within protein-protein complexes;
+- SRVs, for mining mutation [$$$doesnt_need_to_be_pathology_related] phenotypes within protein structures.
 
-The graphs can also be mapped to volumetric grids (i.e., 3D image-like representations). Then the physico-chemical and geometrical features for the grids and/or graphs are computed. The user can choose which features to generate from several ones already defined in the package, and can also define custom features modules, as explained in the documentation. Examples of pre-defined node features are the type of the amino acid, its polarity, the solvent-accessible surface area. Examples of pre-defined edge features are distance, covalent bond, electrostatic potential. The full and detailed list of features can be found in the [documentation's features page](https://deeprank2.readthedocs.io/en/latest/features.html). Multiple CPUs can be used to parallelize and speed up the featurization process. The mapped data are finally saved into HDF5 files, designed to efficiently store and organize big data. Users can then use the data saved into HDF5 files for whatever architecture and DL framework is more suited for the application. In particular, graphs can be used for the training of GNNs, and 3D grids can be used for the training of CNNs.
+Graphs can either be used directly or mapped to volumetric grids (i.e., 3D image-like representations). Then the physico-chemical and geometrical features for the grids and/or graphs are computed [$$$arent_these_computed_before_mapping?] and assigned to each node and edge. The user can choose which features to generate from several pre-existing options defined in the package, or define custom features modules, as explained in the documentation. Examples of pre-defined node features are the type of the amino acid, its size and polarity, as well as more complex features such as its buried surface area and secondary structure features. Examples of pre-defined edge features are distance, covalency, and potential energy. A detailed list of predefined features can be found in the [documentation's features page](https://deeprank2.readthedocs.io/en/latest/features.html). Multiple CPUs can be used to parallelize and speed up the featurization process. The mapped [$$$doesnt_mapped_specifically_refer_to_grids?] data are saved into HDF5 files, designed to efficiently store and organize big data. Users can then use the data for any ML or DL framework suited for the application. Specifically, graphs can be used for the training of GNNs, and 3D grids can be used for the training of CNNs.
```

Comment:
$$$arent_these_computed_before_mapping?
Nope, first there is the graph definition and only after there is the featurization

Reply:
$$$doesnt_mapped_specifically_refer_to_grids?
In a way, data are mapped to both grids and graphs. We can say the "processed" data, maybe? To avoid confusion

Reply:
yes, i think processed is clearer. Especially because we have code using "mapping" terminology which specifically is for converting graphs to grids.